SimG4Core/Application/src/OscarMTMasterThread.cc

```diff
@@ -57,27 +57,28 @@ OscarMTMasterThread::OscarMTMasterThread(const edm::ParameterSet& iConfig)
     runManagerMaster = std::make_shared<RunManagerMT>(iConfig);
     m_runManagerMaster = runManagerMaster;
 
-    edm::LogVerbatim("SimG4CoreApplication") << "OscarMTMasterThread: initialization of RunManagerMT finished";
+    LogDebug("SimG4CoreApplication") << "OscarMTMasterThread: initialization of RunManagerMT finished";
```

Comment:
here the level of verbosity is raised, but I think it can be ok, and make simpler to follow the initialization behaviour in case, for normal operations we filter out this level anyway